package.json

```diff
@@ -1,20 +1,25 @@
 {
-  "name": "suiter",
-  "version": "1.0.0",
+  "name": "element-breakpoints",
+  "version": "0.5.0",
```

Comment:
Not a huge deal, but it's recommended by npm to start all package at `1.0.0` if following semver.